app/pages/analyst/application-review.tsx

```diff
@@ -143,29 +173,19 @@ class ApplicationReview extends Component<Props, State> {
              offset-md-${this.state.isSidebarOpened ? 0 : 1}
              offset-lg-${this.state.isSidebarOpened ? 0 : 1}`}
           >
-            <h1>
-              {`Application #${query.application.rowId}`}
-              <ApplicationRevisionStatusContainer
-                applicationRevisionStatus={
-                  query.application.reviewRevisionStatus
-                }
-                applicationRowId={query.application.rowId}
-              />
-            </h1>
+            <h1>{`Application #${query.application.rowId}`}</h1>
```

Comment:
I do vaguely recall discussing the version number, and it would be quick to add. 👍 

Since the facility info has been moved to the top of the form, it's probably unnecessary to add the the BCGHG ID as it's now literally a couple inches below here (a part of that other discussion about moving the facility info to the top).

Reply:
The BC GHG ID is being removed from the form (see https://youtrack.button.is/issue/GGIRCS-2375), so it would be useful to add it here

Reply:
Oh, just saw your in-progress work on [GGIRCS-2375](https://youtrack.button.is/issue/GGIRCS-2375), which I suppose means there will be nowhere else they can see the BCGHG ID.

I can add that in as well.